guides/source/association_basics.md

```diff
@@ -1227,25 +1449,15 @@ end
 NOTE: You only need to specify the `:counter_cache` option on the `belongs_to`
 side of the association.
 
-Starting to use counter caches on existing large tables can be troublesome, because the column
-values must be backfilled separately of the column addition (to not lock the table for too long)
-and before the use of `:counter_cache` (otherwise methods like `size`/`any?`/etc, which use
-counter caches internally, can produce incorrect results). To safely backfill the values while
-keeping counter cache columns updated with the child records creation/removal and to avoid the
-mentioned methods use the possibly incorrect counter cache column values and always get the results
-from the database, use `counter_cache: { active: false }`. If you also need to specify a custom
-column name, use `counter_cache: { active: false, column: :my_custom_counter }`.
+Using counter caches on existing large tables can be troublesome. To avoid locking the table for too long, the column values must be backfilled separately from the column addition. This backfill must also happen before using `:counter_cache`; otherwise, methods like `size`, `any?`, etc., which rely on counter caches, may return incorrect results.
```

Comment:
The whole `counter_cache` section might deserve it's own section (maybe under tips and tricks?).
Right now it's not even showing up in the sidebar and this is the only guide that mentions them.